default_metadata/string/error.postcode.json

```diff
@@ -3,5 +3,5 @@
   "_type": "string.error",
   "description": "Postcode format is not valid",
   "value:en": "Enter a valid UK postcode for \"%{control}\"",
-  "value:cy": "Nodwch god post y DU dilys ar gyfer \"%{control}\""
+  "value:cy": "Nodwch god post dilys y DU dilys ar gyfer \"%{control}\""
```

Comment:
Wasn't sure of the intention of the correction because it's not worded as a full replacement, but google translate thinks this sentence makes sense with or without the added `dilys`